src/main/resources/assets/computercraft/lua/rom/programs/delete.lua

```diff
@@ -1,16 +1,17 @@
+local args = table.pack(...)
 
-local tArgs = { ... }
-if #tArgs < 1 then
-    print( "Usage: rm <path>" )
+if args.n < 1 then
+    print("Usage: rm <paths>")
     return
 end
 
-local sPath = shell.resolve( tArgs[1] )
-local tFiles = fs.find( sPath )
-if #tFiles > 0 then
-    for n,sFile in ipairs( tFiles ) do
-        fs.delete( sFile )
+for i = 1, args.n do
+    local files = fs.find(shell.resolve(args[i]))
+    if #files > 0 then
+        for n, file in ipairs(files) do
+            fs.delete(file)
+        end
+    else
+        printError(args[i] .. ": No matching files")
```

Comment:
Maybe `No matching files for '%s'`? Not sure, but probably should wrap it in quotes either way.

Reply:
well, your call

Reply:
can I get a formatting printError though?